pkg/spdk/backup.go

```diff
@@ -225,7 +225,7 @@ func (b *Backup) CloseSnapshot(snapshotName, volumeName string) error {
 	}
 
 	b.log.Info("Stopping NVMe initiator")
-	if _, err := b.initiator.Stop(false, true); err != nil {
+	if _, err := b.initiator.Stop(false, false, true); err != nil {
```

Comment:
`false, true, true`? the second parameter should be `isDeferred`?

Reply:
Either `false, true, true` or `false, false, true` is fine, because backup doesn't use dm linear device.